deploy/helm/kubecf/assets/operations/instance_groups/app_autoscaler.yaml

```diff
@@ -0,0 +1,1150 @@
+            server_key:  ((apiserver_server.private_key))
+            scheduler:
+              ca_cert: ((scheduler_ca.ca))
+              client_cert: ((scheduler_client.certificate))
+              client_key: ((scheduler_client.private_key))
+            scaling_engine:
+              ca_cert: ((scalingengine_ca.ca))
+              client_cert: ((scalingengine_client.certificate))
+              client_key: ((scalingengine_client.private_key))
+            metrics_collector:
+              host: metricsserver.service.cf.internal
+              port: *metricsserverPort
+              ca_cert: ((metricsserver_ca.ca))
+              client_cert: ((metricsserver_client.certificate))
+              client_key: ((metricsserver_client.private_key))
+            eventgenerator:
+              ca_cert: ((eventgenerator_ca.ca))
+              client_cert: ((eventgenerator_client.certificate))
+              client_key: ((eventgenerator_client.private_key))               
+            service_broker:
+              ca_cert: ((servicebroker_ca.ca))
+              client_cert: ((servicebroker_client.certificate))
+              client_key: ((servicebroker_client.private_key))
+          policy_db: *database
+          cf: *cf_credentials
+    - name: servicebroker
+      release: app-autoscaler
+      properties:
+        autoscaler:
+          service_broker: 
+            enable_custom_metrics: true
+            db_config: *db_config
+            publicPort : &servicebrokerPort 6101
+            port: 6107
+            health:
+              port: 6201
+            ca_cert: ((servicebroker_ca.ca))
+            server_cert: ((servicebroker_server.certificate))
+            server_key: ((servicebroker_server.private_key))
+            public_ca_cert: ((servicebroker_public_ca.ca))
+            public_server_cert: ((servicebroker_public_server.certificate))
+            public_server_key: ((servicebroker_public_server.private_key))
+            username: autoscaler_service_broker_user
+            password: ((autoscaler_service_broker_password))
+            http_request_timeout: 5000
+            dashboard_redirect_uri: ""
+            catalog: 
+              services:
+              - id: autoscaler-guid
+                name: autoscaler
+                description: Automatically increase or decrease the number of application instances based on a policy you define.
+                bindable: true
+                plans:
+                - id: autoscaler-free-plan-id
+                  name: autoscaler-free-plan
+                  description: This is the free service plan for the Auto-Scaling service.
+            api_server:
+              ca_cert: ((apiserver_ca.ca))
+              client_cert: ((apiserver_client.certificate))
+              client_key: ((apiserver_client.private_key))
+          binding_db: *database
+    - name: route_registrar
+      release: routing
+      consumes:
+        nats: {from: nats, deployment: cf}
```

Comment:
This can use the YAML syntax. Also, `deployment` should be `{{ .Values.deployment_name | quote }}`.

Reply:
This was changed to  
``` 
consumes:
  nats:
    from: nats
    deployment: {{ .Values.deployment_name | quote }} 
```